cmd/release/cmd/stats.go

```diff
@@ -0,0 +1,123 @@
+package cmd
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"sort"
+	"time"
+
+	"github.com/briandowns/spinner"
+	"github.com/google/go-github/v39/github"
+	"github.com/rancher/ecm-distro-tools/repository"
+	"github.com/spf13/cobra"
+)
+
+var (
+	repo      *string
+	startDate *string
+	endDate   *string
+)
+
+const layout = "2006-01-02"
+
+var repoToOwner = map[string]string{
+	"rke2":    "rancher",
+	"rancher": "rancher",
+	"k3s":     "k3s-io",
+}
+
+// statsCmd represents the stats command
+var statsCmd = &cobra.Command{
+	Use:   "stats",
+	Short: "Release statistics",
+	Long:  ``,
+	Run: func(cmd *cobra.Command, args []string) {
+		from, err := time.Parse(layout, *startDate)
+		if err != nil {
+			fmt.Fprint(os.Stderr, err)
+			os.Exit(1)
+		}
+		to, err := time.Parse(layout, *endDate)
+		if err != nil {
+			fmt.Fprint(os.Stderr, err)
+			os.Exit(1)
+		}
+		githubToken := os.Getenv("GITHUB_TOKEN")
+
+		ctx := context.Background()
+		client := repository.NewGithub(ctx, githubToken)
```

Comment:
This command doesn't require anything in the config file and I'm going to be adding another PR to reflect that since not all commands need that file and the convention is to have the token in an env var.